README.md

```diff
@@ -1,37 +1,53 @@
 ## Coq LSP
 
-The `coq-lsp` project aims to provide a lightweight, pure [Language Server
-Protocol](https://microsoft.github.io/language-server-protocol/)
-server implementation for the Coq proof assistant, as well as to serve
-as a framework for interface experimentation.
+`coq-lsp` is a new [Language
```

Comment:
```suggestion
`coq-lsp` is a [Language
```

Reply:
I did this change 3 times then reverted it back XD but ok your vote removes the tie :)